src/components/Header/__tests__/Header.spec.tsx

```diff
@@ -1,33 +1,42 @@
 import React from "react";
 import { Header } from "../Header";
 import { fireEvent, render, screen } from "@testing-library/react";
-import { ThemeContext } from "../../ThemeSetter";
+import { ThemeContext } from "../../ThemeContext";
 
-describe("testing Header", () => {
-  it("should render Header", () => {
+describe("Header", () => {
+  it("Header Match snapshot test", () => {
```

Comment:
word "test" shouldn't be there in explanation.

Reply:
okay